checks/shell_download_validate.go

```diff
@@ -35,6 +35,10 @@ var (
 	shellNames = []string{
 		"sh", "bash", "dash", "ksh", "mksh",
 	}
+	// supportedShells is the list of shells that are supported by mvdan.cc/sh/v3/syntax.
```

Comment:
you can declare `supportedShells `, and build `shellNames` from it. Take `shellInterpreters` as an example.

Reply:
Good idea.